enterprise/server/raft/filestore/filestore.go

```diff
@@ -227,7 +244,32 @@ func (pmk *PebbleKey) Bytes(version PebbleKeyVersion) ([]byte, error) {
 		partDir := PartitionDirectoryPrefix + pmk.partID
 		filePath = filepath.Join(partDir, filePath, "v4")
 		return []byte(filePath), nil
+	case Version5:
+		if len(pmk.fullKey) > 0 {
+			return pmk.fullKey, nil
+		}
+		hashStr := pmk.hash
+		if pmk.isolation == "ac" {
+			hashExtra := remapANONToFixedGroupID(FixedWidthGroupID(pmk.groupID))
```

Comment:
nit: you can skip the FixedWidthGroupID call as it doesn't matter for hashing (and corresponding trimFixedWidthGroupID call)

Reply:
Done.